server.js

```diff
@@ -1,11 +1,59 @@
-var path = require('path');
-var express = require('express');
+const express = require('express');
+const path = require('path');
+const gonebusy = require('gonebusy-nodejs-client');
+const Promise = require('bluebird').Promise;
+const url = require('url');
 
-var app = express();
+gonebusy.configuration.BASEURI = 'https://sandbox.gonebusy.com/api/v1';
+const authorization = 'Token ad86866858ea7c256390f96f58c520df';
+const serviceId = 2010395226;
 
-var staticPath = path.join(__dirname, '/public');
-app.use(express.static(staticPath));
+const services = Promise.promisifyAll(gonebusy.ServicesController);
+const resources = Promise.promisifyAll(gonebusy.ResourcesController);
 
-app.listen(4000, function() {
+
+const app = express();
+
+if (process.env.NODE_ENV === 'production') {
+    var staticPath = path.join(__dirname, '/public');
+    app.use(express.static(staticPath));
+}
+
+app.get('/services/resources', (req, res) => {
```

Comment:
oh wait I remember why I did `/services/resources` because I had this endpoint only return resources. Should I have it return the entire service instead?